src/ostree/ot-builtin-summary.c

```diff
@@ -107,6 +107,80 @@ ostree_builtin_summary (int argc, char **argv, GCancellable *cancellable, GError
             return FALSE;
         }
 
+#ifdef OSTREE_ENABLE_EXPERIMENTAL_API
+      const char *collection_id = ostree_repo_get_collection_id (repo);
+#else  /* if !OSTREE_ENABLE_EXPERIMENTAL_API */
+      const char *collection_id = NULL;
+#endif  /* OSTREE_ENABLE_EXPERIMENTAL_API */
+
+      /* Write out a new metadata commit for the repository. */
```

Comment:
Can you explain a bit *what* metadata is being included here?  It looks it's basically a stub, but callers could use `--add-metadata`?

Reply:
`additional_metadata` is passed into the vardict for `new_summary_commit`, so the stored metadata is identical to that in the `summary` file (plus the collection/ref binding keys). By default, that is indeed nothing, unless the user provides an `--add-metadata` option.

Reply:
Hmm, but if the intention is to get rid of the summary file down the line, don't we want the list of refs/deltas?

Reply:
The intention is not to get rid of the summary file down the line, since it remains a useful cache for the list of refs/deltas. The `ostree-metadata` ref is only replacing the *additional metadata* from the `summary`.